### ISSUE TITLE
Feature Request: Open in new tab

### ISSUE THREAD

**Issue by Giltvedt** [Quorafind/Obsidian-Float-Search#30]
### Feature Requested

Plugins is powerful and flexible with key combination while Float Search modal is active. Now I prefer to use this instead search to lookup my notes for specific word/text.

I'm working with multiple notes and miss the opportunity to open in new tab instead open it on active tab.

It's a workaround on this when I have to right-click with mouse to open "Open in new tab". Would be great to add this as key combination such cmd (macOS) or ctrl (windows/linux) with enter-button. To open note in new tab.

Thank you for this great plugin!

### Relevant Screenshot

![Obsidian_Float-Search_feature-request_Open-in-new-tab](https://github.com/Quorafind/Obsidian-Float-Search/assets/965716/cd508753-7caa-4a61-ade5-c22af901fc7e)

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by Quorafind** [comment]
I remember that I have added this function in early version.

**Comment by Giltvedt** [comment]
@Quorafind Thank you for response.

This was strange. Until now I have not been able to open it in a new tab. I did something small changes with my setup, with unrelated plugins, before I saw your comment. Tested it with cmd+enter now and it actually worked as it should.

I posted this because there was no alternative among these key combinations. Which gave the impression that there was no such key combination. Check the picture in the attachment.

Then there was something with my setup that either conflicted or blocked this key combination, until now. In that case, it was unnecessary to create this Feature Request. Sorry for the unnecessary.

Again I want to say that this tool is truly superb!

![Obsidian_Float-Search_feature-request_Open-in-new-tab_list](https://github.com/Quorafind/Obsidian-Float-Search/assets/965716/394f64db-8fe1-40c8-a8e7-65f1046aafd8)
